### ISSUE TITLE
Wither Storm Regeneration in the config

### ISSUE THREAD

**Issue by ghost** [nonamecrackers2/crackers-wither-storm-mod#932]
In my opinion a coll addition could be an option called "Wither Storm Regeneration" which would make it so it heals if it eats something so it could reach full health in later phases after evolving even with the invincibility turned off

**Comment by myron-obsolete-account** [comment]
Doesn't the Wither Storm already regenerate?

**Comment by Globalfirestorm10** [comment]
Yeah, i'm pretty sure it already does via block clusters (and blocks in earlier stages)
